.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -1,5 +1,8 @@
 name: Deploy to Azure App Service
 
+concurrency:
+    group: AzureDeployApp
```

Comment:
Added concurrency group restriction to every workflow that modify Azure resources. Only the github context is available here, so we can't make it dynamic based on the parameters passed to the workflow.